web/src/.eslintrc.js

```diff
@@ -47,12 +44,21 @@ module.exports = {
         ignoreRestSiblings: true,
       },
     ],
+    'no-use-before-define': 'off',
```

Comment:
I believe we left this rule on explicitly to enforce code consistency between any contributors

Reply:
Ah interesting, I didn't know that so thanks for the docs link 👍